src/renderer/components/dock/terminal/terminal-fonts.injectable.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { getInjectable } from "@ogre-tools/injectable";
+import { beforeFrameStartsFirstInjectionToken } from "../../../before-frame-starts/tokens";
+import RobotoMono from "../../../fonts/Roboto-Mono-nerd.ttf"; // patched font with icons
+import AnonymousPro from "../../../fonts/AnonymousPro-Regular.ttf";
+import IBMPlexMono from "../../../fonts/IBMPlexMono-Regular.ttf";
+import JetBrainsMono from "../../../fonts/JetBrainsMono-Regular.ttf";
+import RedHatMono from "../../../fonts/RedHatMono-Regular.ttf";
+import SourceCodePro from "../../../fonts/SourceCodePro-Regular.ttf";
+import SpaceMono from "../../../fonts/SpaceMono-Regular.ttf";
+import UbuntuMono from "../../../fonts/UbuntuMono-Regular.ttf";
+
+export const terminalFontsInjectable = getInjectable({
+  id: "terminalFontsInjectable",
+
+  instantiate() {
+    return new Map([
```

Comment:
nit: either this map should be created from a call to `di.injectMany(terminalFontInjectionToken)` (for some new injection token) and then be fully OCP or it doesn't need to be wrapped in an injectable.